lib/shared/store.js

```diff
@@ -432,24 +443,39 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         })
       }
 
-      const selectedDrive = findDrive(state, state.getIn([ 'selection', 'drive' ]))
+      const selectedDevices = state.getIn([ 'selection', 'devices' ])
```

Comment:
Line 350 (and line 475) has `const selectionDevices` but here you have `const selectedDevices`
(IMHO the latter sounds better)

Reply:
I've renamed them `selectedDevices` now